src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/AbstractEntityValidation.feature.disabled

```diff
@@ -0,0 +1,19 @@
+Feature: Client cannot POST an abstract entity
+  Ensure that clients cannot POST an abstract entity
+
+  @ignore
+  Scenario: Validation of an Abstract Entity
+    Clients cannot POST to the Education Organization abstract entity
+    When sending a POST request to "/ed-fi/educationOrganization" with body
+            """
+            """
+    Then it should respond with 404
+
+  @ignore
+  Scenario: Ensure that clients cannot POST an abstract entity (Student Program Association)
+    Clients cannot POST to the Student Program Association abstract entity
+    When sending a POST request to "/ed-fi/studentProgramAssociations" with body
+            """
+            """
+    Then the response code is 404
+
```

Comment:
Please also add similar scenarios for both abstract entities, covering GET requests: GET "/ed-fi/educationOrganizations"

We do not need to cover DELETE and PUT: since the GET request does not return any results and the POST doesn't work, there is no way to have a resource that could be updated or deleted.

Reply:
Done